### PR TITLE
Update county Baltimore

### PR DESCRIPTION
**Pull request by nguyenhoan1988**
Updating `data/102/081/589/102081589.geojson` using the [WOF Editor](https://github.com/iandees/wof-editor)

### CODE REVIEW

**Comment by stepps00** [comment]
Thanks for the change suggestion @nguyenhoan1988 

I do see that Who's On First has this Baltimore locality record: https://spelunker.whosonfirst.org/id/85949461, but know that label properties on counties were intentionally set. cc @nvkelso any thoughts here? I see this change was made 5+ years ago, but not sure what the reasoning was to set county to "City of Baltimore."

**Comment by nvkelso** [comment]
**tl;dr** the current setup is as intended, we should close this PR (not merge it). If additional action is needed, it should be to link up the locality and county records as coterminous to each other.

**Longer story:** So Baltimore is weird... There's the "city" of Baltimore around the harbour and the "county" of Baltimore around the city. Both the city (locality) and the county (county) are both county-level equivalents per Maryland statues. In WOF this means the city should have a locality record [85949461](https://spelunker.whosonfirst.org/id/85949461) and a coterminous county record (this one [102081589](https://spelunker.whosonfirst.org/id/102081589)).

**Comment by nvkelso** [comment]
@nguyenhoan1988  if you can tell us more about your use case / how this "bug" is manifesting for you we could provide some alternatives?
